[2025-05-08T16:51:27.690-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T16:51:27.696-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T21:50:40+00:00 [queued]>
[2025-05-08T16:51:27.701-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T21:50:40+00:00 [queued]>
[2025-05-08T16:51:27.701-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-08T16:51:27.716-0500] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): modoOperacion> on 2025-05-08 21:50:40+00:00
[2025-05-08T16:51:27.721-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'request_operation', 'modoOperacion', 'scheduled__2025-05-08T21:50:40+00:00', '--job-id', '2650', '--raw', '--subdir', 'DAGS_FOLDER/operations.py', '--cfg-path', '/tmp/tmp_ffxpilc']
[2025-05-08T16:51:27.722-0500] {standard_task_runner.py:105} INFO - Job 2650: Subtask modoOperacion
[2025-05-08T16:51:27.724-0500] {logging_mixin.py:190} WARNING - /home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=132661) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-08T16:51:27.724-0500] {standard_task_runner.py:72} INFO - Started process 132692 to run task
[2025-05-08T16:51:27.751-0500] {task_command.py:467} INFO - Running <TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T21:50:40+00:00 [running]> on host DESKTOP-0R7MK50.
[2025-05-08T16:51:27.803-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='request_operation' AIRFLOW_CTX_TASK_ID='modoOperacion' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T21:50:40+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T21:50:40+00:00'
[2025-05-08T16:51:27.804-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T16:51:29.062-0500] {logging_mixin.py:190} WARNING - /home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/urllib3/connectionpool.py:1097 InsecureRequestWarning: Unverified HTTPS request is being made to host 'auth.iqoption.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
[2025-05-08T16:51:31.069-0500] {python.py:240} INFO - Done. Returned value was: Modo PRACTICE. Balance actual: 9937.2
[2025-05-08T16:51:31.084-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T16:51:31.084-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=request_operation, task_id=modoOperacion, run_id=scheduled__2025-05-08T21:50:40+00:00, execution_date=20250508T215040, start_date=20250508T215127, end_date=20250508T215131
[2025-05-08T16:51:31.143-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-08T16:51:31.152-0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-08T16:51:31.153-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
